Dockerfile

```
@@ -1,6 +1,7 @@
 # Build the manager binary
 ARG GOLANG_VERSION=1.18.4
 FROM registry.access.redhat.com/ubi8/go-toolset:$GOLANG_VERSION as builder
```

```suggestion
ARG LOCAL_BUNDLE=odh-manifests.tar.gz
FROM registry.access.redhat.com/ubi8/go-toolset:$GOLANG_VERSION as builder
```

I forgot to add the default initialization of `LOCAL_BUNDLE` needs to occur at the global scope so that the other stages can inherit the default value.